### PR TITLE
Exercicio 5

### PR DESCRIPTION
**Pull request by thiagopinheirolima**

### CODE REVIEW

File: Exercicio 5/BooleanToVisibilityConverter.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Data;
+
+namespace Exercicio_5
+{
+    class BooleanToVisibilityConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, string language)
+        {
+
```
Comment (soutolucas) [inline Exercicio 5/BooleanToVisibilityConverter.cs]: Could you please remove this line?

File: Exercicio 5/BooleanToVisibilityConverter.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+            return (bool)value ? Visibility.Visible : Visibility.Collapsed;
+        }
+
+        public object ConvertBack(object value, Type targetType, object parameter, string language)
+        {
+            throw new NotImplementedException();
```
Comment (soutolucas) [inline Exercicio 5/BooleanToVisibilityConverter.cs]: Could you please implement ConvertBack method?

File: Exercicio 5/BooleanToVisibilityConverter.cs [hunk 3]
```diff
@@ -12,13 +12,12 @@ class BooleanToVisibilityConverter : IValueConverter
...
         {
-
-            return (bool)value ? Visibility.Visible : Visibility.Collapsed;
+            return (bool) value ? Visibility.Visible : Visibility.Collapsed;
         }
 
         public object ConvertBack(object value, Type targetType, object parameter, string language)
         {
-            throw new NotImplementedException();
+            return (Visibility) value == Visibility.Visible ? true : false;
```
Comment (soutolucas) [inline Exercicio 5/BooleanToVisibilityConverter.cs]: Please, remove "? true : false" it's not necessary.

Comment (thiagopinheirolima) [reply Exercicio 5/BooleanToVisibilityConverter.cs]: Done

**Pull request merged**